skin.json

```diff
@@ -40,8 +40,23 @@
 		},
 		"ChameleonEnableExternalLinkIcons": {
 			"value": false
+		},
+		"ChameleonStickyTOCFloat": {
```

Comment:
I don't think we should have component config in here. All other components do this in the layout file and I don't see a special reason for this to be an exception.